pkg/cmd/pr/merge/merge.go

```diff
@@ -47,7 +47,7 @@ type MergeOptions struct {
 	UseAdmin                bool
 	IsDeleteBranchIndicated bool
 	CanDeleteLocalBranch    bool
-	InteractiveMode         bool
+	MergeStrategyEmpty      bool
```

Comment:
InteractiveMode in this command meant the user did not provide a merge strategy, and `gh` should prompt for it. This renaming makes that clearer.